### PR TITLE
ceph-pr-docs: install python3 packages

### PR DESCRIPTION
**Pull request by tchaikov**
see also https://github.com/ceph/ceph/pull/29528

Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

**Comment by alfredodeza** [comment]
You might want to add `python3` to the list of labels needed for this to build. If we ever start cleaning up the labels in Jenkins, this will break otherwise

File: ceph-pr-docs/build/build [hunk 1]
```diff
@@ -9,6 +9,6 @@ sudo rm -f /etc/apt/sources.list.d/ubuntu-toolchain-r*
 
 # Ceph doc build deps, Ubuntu only because ditaa is not packaged for CentOS
 sudo apt-get update -o Acquire::Languages=none -o Acquire::Translation=none
-sudo apt-get install -y gcc python-dev python-pip python-virtualenv libxml2-dev libxslt-dev doxygen graphviz ant ditaa cython
+sudo apt-get install -y gcc python3-dev python3-pip python3-virtualenv libxml2-dev libxslt-dev doxygen graphviz ant ditaa cython3
```
Comment (alfredodeza) [inline ceph-pr-docs/build/build]: have you checked that these are the right package names? I see that cython is cython3, so you probably have :)

Comment (tchaikov) [reply ceph-pr-docs/build/build]: yeah, https://github.com/ceph/ceph/pull/29528 was tested with this change.

**Comment by smithfarm** [comment]
This appears to have broken the `docs: build check` for PRs targeting mimic. See https://github.com/ceph/ceph/pull/29766

**Pull request merged**